client/components/AccountAdmin/UserRows.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Checkbox, Table } from 'semantic-ui-react';
+
+const USER_ROLES = Object.freeze({
+    super_admin: 'Super Admin',
+    admin: 'Admin',
+    researcher: 'Researcher',
+    facilitator: 'Facilitator',
+    participant: 'Participant'
+});
+
+const onCheckboxClick = async (event, { email, username, role, checked }) => {
+    const endpoint = checked ? '/api/roles/add' : '/api/roles/delete';
+    const body = JSON.stringify({
+        roles: [role],
+        email,
```

Comment:
What happens if there is no value for `email`? `email` is not required for creating an account.

Reply:
Currently nothing happens, it is just undefined in this req body and instead we'll use the username to get the right user to edit their roles. I have fixed/re-added the "required" attribute on the username field for account creation, so there will at least always be a username.